### PR TITLE
fonttools

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

**Comment by CJ-Wright** [comment]
Pending https://github.com/conda-forge/brotli-feedstock/pull/8

File: recipes/fonttools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    - unicodedata2
+    - munkres
+
+test:
+  imports:
+    - fonttools
+
+about:
+  home: https://github.com/fonttools/fonttools
+  license: BSD-style
```
Comment (synapticarbors) [inline recipes/fonttools/meta.yaml]: The `setup.py` claims this as BSD, but it looks like an old variant of the MIT license:

https://fedoraproject.org/wiki/Licensing:MIT#Old_Style_with_legal_disclaimer

Not sure if we should really change anything here. Just wanted to point it out.

Comment (CJ-Wright) [reply recipes/fonttools/meta.yaml]: Thank you! I was rather confused by that.

Comment (synapticarbors) [reply recipes/fonttools/meta.yaml]: That said, I guess it's preferable to label the license as the authors of the package did. I wouldn't change this to `MIT`. It is confusing though. Maybe raise an issue seeking clarification upstream?

Comment (CJ-Wright) [reply recipes/fonttools/meta.yaml]: for reference https://github.com/fonttools/fonttools/issues/1127

**Pull request merged**